### ISSUE TITLE
{valueY.percent.formatNumber('#.0')} displays nothing when value = 0

### ISSUE THREAD

**Issue by morrow95** [amcharts/amcharts4#1383]
Using the following in a chart : 

```
series.calculatePercent = true;
series.tooltipText = "Total : [bold]{valueY}[/]\nPercent : [bold]{valueY.percent.formatNumber('#.0')}%[/]";
```

When the value is 0 it shows as the following in the chart : 
```
Total : 0
Percent : %
```

Either `percent` or `formatNumber` or both should be corrected to handle a 0 value being passed in.  Expected result would be : 
```
Total : 0
Percent : 0.0%
```

**Comment by martynasma** [comment]
Hm, do you have an example of this misbehavior? I can't seem to be able to reproduce it:

![image](https://user-images.githubusercontent.com/1458520/59585379-00fef780-90e9-11e9-9df1-9235e466d81e.png)

https://codepen.io/team/amcharts/pen/2bd68f715161ac48381b20a43aeb175a?editors=0010

**Comment by morrow95** [comment]
Example - [https://jsfiddle.net/qL9hpzb2/](https://jsfiddle.net/qL9hpzb2/)

**Comment by martynasma** [comment]
OK, so you have data with all zero values. How does one calculate percentage of zero out of zero? :)

**Comment by morrow95** [comment]
The results come from the database, so yes, it is possible to have zero values (no results).  My point was this should be handled as 0% for this reason :)

**Comment by martynasma** [comment]
Fixed in [4.5.3](https://github.com/amcharts/amcharts4/releases/tag/4.5.3).

## [4.5.3] - 2019-06-18

### Added
- `"up"`, `"down"`, `"left"` and `"right"` options added to `Tooltip.pointerOrientation`. Allows forcing constant position/orientation of the Tooltip. Note, the tooltip might go out of div/screen boundaries when using these new orientations.
- New property added to `SliceGrouper`: `groupProperties`. It's an object comprising by key/value pairs to apply to a "group slice", e.g. `grouper.groupProperties.fill = am4core.color("red")`.
- It was impossible to reuse single pattern for multiple `Sprite` elements, as disposing one was resulting in pattern to be disposed as well.

### Fixed
- `hideOversized` was being ignored for HTML-based labels.
- `PieChart` series' `legendSettings` were shared across all series; it was impossible to set different `valueText`.
- `CategoryAxis` with HTML labels could in result excessive white space at the bottom of the chart.
- In some cases patterns set as `Sprite`'s fills using `fill` adapter were not displayed.
- When calculating totals on a Series with all zero values, `percent` was not being calculated. It will now have a zero.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
